src/components/PagedTable.js

```diff
@@ -22,6 +22,7 @@ const PagedTable = ({ pageSize, rows, children, ...props }) => {
           <tbody>{displayRows}</tbody>
         </Table>
       </div>
+      {validPages.length > 1 && <Paginate count={validPages.length} active={current} onClick={(i) => setPage(i)} />}
```

Comment:
Just flagging this, as it's a change to how `PagedTable` is displayed and is adjacent to the main purpose of the PR.